Package.swift

```diff
@@ -37,14 +40,21 @@ let package = Package(
       name: "CTopazRuntime",
       dependencies: []),
 
+    .target(
+      name: "CTopazLib",
```

Comment:
👍 for creating a new target for this purpose. Better than mixing it with the runtime library.